packages/connect-voting-disputable/src/__test__/models/votes.test.ts

```diff
@@ -1,13 +1,21 @@
-import { DisputableVoting, Vote, CastVote, DisputableVotingConnectorTheGraph } from '../../../src'
+import {
```

Comment:
I notice we had several lint corrections on a few of the files. What you think of adding workflow with a tool like husky to improve the developer practice and only allow push after linting?

Reply:
Yes good idea! I am going to add this as a separate PR.

Reply:
https://github.com/aragon/connect/pull/207